docs/basics.rst

```diff
@@ -448,7 +448,7 @@ Validating a Form Submission
 ----------------------------
 
 Once the user seen the form and has chewed on its inputs a bit, he
-will eventually submit the form.  When he submits it, the logic you
+will eventually submit the form.  When they submit it, the logic you
```

Comment:
I'd rather rewrite to remove the redundancy, e.g.:

    will eventually submit the form.  At that point, the logic you

(Again, note the `he` at the end of the previous line).